app/views/users/index.js.erb

```diff
@@ -0,0 +1 @@
+$('#UsersList').html("<%= escape_javascript(render 'user_list') %>");
```

Comment:
I was about to suggest `<%= j render 'user_list' %>` as a cleaner way to do the same thing as `escape_javascript(...)` but that's just excessive and obnoxious. ;-)